### PR TITLE
feat: Create markdown html extractor

### PR DESCRIPTION
**Pull request by rodolfocoding**

### CODE REVIEW

File: Projects/2-Intermediate/Html-Extractor-App.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Html Extractor
```
Comment (laisfrigerio) [inline Projects/2-Intermediate/Html-Extractor-App.md:1]: Hey, @rodolfocoding 

Please, add the reference for this exercise on the README.md file!!!

**Review by laisfrigerio — CHANGES_REQUESTED** [review]

**Comment by laisfrigerio** [comment]
Hey, @rodolfocoding 

First of all, Thanks for your contribution 🚀

I made a little suggestion on the review above  ☝️

File: Projects/2-Intermediate/Html-Extractor-App.md (hunk 1 above)
Comment (rodolfocoding) [reply Projects/2-Intermediate/Html-Extractor-App.md:1]: Change made :)

**Review by laisfrigerio — APPROVED** [review]

**Comment by laisfrigerio** [comment]
Greate Job, @rodolfocoding 

Thanks for your contribution 🚀

**Pull request merged**